crates/wasmtime/src/engine.rs

```diff
@@ -61,6 +63,8 @@ struct EngineInner {
     signatures: TypeRegistry,
     #[cfg(feature = "runtime")]
     epoch: AtomicU64,
+    #[cfg(feature = "runtime")]
+    custom_code_memory: Option<Arc<dyn CustomCodeMemory>>,
```

Comment:
This is already stored as-is in `Config` so might be able to access from there (stored above) instead of also storing here too?

Reply:
Ah, right, good point -- updated.